doc/releases/migration-guide-3.6.rst

```diff
@@ -19,6 +19,8 @@ Boards
 
 * The deprecated Nordic SoC Kconfig option ``NRF_STORE_REBOOT_TYPE_GPREGRET`` has been removed,
   applications that use this should switch to using the :ref:`boot_mode_api` instead.
+* NXP: Enabled :ref:`linkserver<linkserver-debug-host-tools>` to be the default runner,
+  if none was set in the board's ``board.cmake`` file.
```

Comment:
I don't understand the last part. The linkserver was made the default no matter what was set before?

Reply:
Rewording but this was set as the default only on NXP boards and not all of them yet. Still a work in progress. Will be clarifying the language here.

Reply:
i think brix is pointing out that currently the line here says "if the board did not have a default runner, we set it to linkserver", and that does not reflect the reality which is actually "we set many nxp boards to default runner linkserver regardless of whether they already had a default"

Reply:
This was a copy paste from the commit. I have updated the description with more information.